tests/baselines/reference/findAllReferencesLinkTag1.baseline.jsonc

```diff
@@ -0,0 +1,875 @@
+
+[
+  {
+    "definition": {
+      "containerKind": "",
+      "containerName": "",
+      "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+      "kind": "method",
+      "name": "(method) C.s(): void",
+      "textSpan": {
+        "start": 43,
+        "length": 1
+      },
+      "displayParts": [
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": "method",
+          "kind": "text"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "C",
+          "kind": "className"
+        },
+        {
+          "text": ".",
+          "kind": "punctuation"
+        },
+        {
+          "text": "s",
+          "kind": "methodName"
+        },
+        {
+          "text": "(",
+          "kind": "punctuation"
+        },
+        {
+          "text": ")",
+          "kind": "punctuation"
+        },
+        {
+          "text": ":",
+          "kind": "punctuation"
+        },
+        {
+          "text": " ",
+          "kind": "space"
+        },
+        {
+          "text": "void",
+          "kind": "keyword"
+        }
+      ],
+      "contextSpan": {
+        "start": 36,
+        "length": 14
+      }
+    },
+    "references": [
+      {
+        "textSpan": {
+          "start": 43,
+          "length": 1
+        },
+        "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+        "contextSpan": {
+          "start": 36,
+          "length": 14
+        },
+        "isWriteAccess": true,
+        "isDefinition": true
+      },
+      {
+        "textSpan": {
+          "start": 492,
+          "length": 1
+        },
+        "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+        "isWriteAccess": false,
+        "isDefinition": false
+      },
+      {
+        "textSpan": {
+          "start": 510,
+          "length": 1
+        },
+        "fileName": "/tests/cases/fourslash/findAllReferencesLinkTag1.ts",
+        "isWriteAccess": false,
+        "isDefinition": false
+      }
+    ]
+  }
+]
+
+// === /tests/cases/fourslash/findAllReferencesLinkTag1.ts ===
+// class C {
+//     m() { }
+//     n = 1
+//     static s() { }
+//     /**
+//      * {@link m}
+//      * @see {m}
+//      * {@link C.m}
+//      * @see {C.m}
+//      * {@link C#m}
+//      * @see {C#m}
+//      * {@link C.prototype.m}
+//      * @see {C.prototype.m}
+//      */
+//     p() { }
+//     /**
+//      * {@link n}
+//      * @see {n}
+//      * {@link C.n}
+//      * @see {C.n}
+//      * {@link C#n}
+//      * @see {C#n}
+//      * {@link C.prototype.n}
+//      * @see {C.prototype.n}
+//      */
+//     q() { }
+//     /**
+//      * {@link s}
+//      * @see {s}
+//      * {@link C.s}
+//      * @see {C.s}
+//      */
+//     r() { }
+// }
+// 
+// interface I {
+//     [|a|]/*FIND ALL REFS*/()
```

Comment:
This one, and the one on `b`, also have no results apart from themselves. Couldn’t tell for sure whether that was expected.

Reply:
I was just lazy and omitted the tests that should start passing in this PR. I added some references that will pass with this PR.